test/EFCore.PG.FunctionalTests/TestUtilities/NpgsqlDbContextOptionsBuilderExtensions.cs

```diff
@@ -1,14 +1,15 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
 using Npgsql.EntityFrameworkCore.PostgreSQL.Infrastructure;
 
 namespace Npgsql.EntityFrameworkCore.PostgreSQL.TestUtilities
 {
-    public static class DbContextOptionsBuilderExtensions
+    public static class NpgsqlDbContextOptionsBuilderExtensions
     {
         public static NpgsqlDbContextOptionsBuilder ApplyConfiguration(this NpgsqlDbContextOptionsBuilder optionsBuilder)
         {
+            //optionsBuilder.ExecutionStrategy(d => new TestNpgsqlRetryingExecutionStrategy(d));
```

Comment:
Why commented?

Reply:
Because this introduces a lot of test breakage which I didn't want to deal with at this point. We should do this at some point.

Reply:
Opened #1244 to track this.